pyproject.toml

```diff
@@ -25,6 +25,7 @@ exasol-data-science-utils-python = {git = "https://github.com/exasol/data-scienc
 click = "^8.0.4"
 pyexasol = "^0.25.0"
 pandas = "^1.1.0"
+password = "^0.2"
```

Comment:
What exactly do yo need this for?

Reply:
good catch, I think, that was something the IDE did by accident

Reply:
will remove